buildSrc/subprojects/performance-testing/src/test/kotlin/gradlebuild/performance/tasks/DetermineBaselinesTest.kt

```diff
@@ -83,6 +86,7 @@ class DetermineBaselinesTest {
 
     @Test
     fun `determines fork point commit on feature branch and empty configuration`() {
+        Assume.assumeFalse(OperatingSystem.current().isWindows)
```

Comment:
Let's leave a comment here about why we need to not run this on Windows.

Reply:
Done.